content/_releases/22/18.md

```diff
@@ -0,0 +1,101 @@
+
+*   In case you don't have 2FA enabled, this is the number of failed login attempts.
+*   In case you have 2FA enabled, this is combined number of failed 2FA code and login attempts. This means if your user has 2FA enabled and succeeded with login but failed with 2FA more than the value of `MAX_LOGIN_ATTEMPTS` then your user will be locked out of the system. To unlock such users the tenant administration must first [disable 2FA for this user]({{site.data.tenant.apiDocsUri}}/v2/#disable-two-factor-authentication-(totp)-for-a-user) and instruct users to navigate `/forgot` address of the tenant to reset their password. This will reset the counter and user can login again.
+
+
+## OEM Related
+
+News in this section are for our customers who use
+[OEM version of the elastic.io platform](https://www.elastic.io/saas-embedded-integration/).
+
+### Removing docker registry builds on component deletion
+
+The platform builds a component docker image every time you push a new version of
+your component. When you remove the component or a particular version of it from
+the platform, an associated docker build would remain until you would remove it at
+later stage directly from the docker registry.
+
+With platform {{page.version}} we improved the situation in the following way:
+*   When you remove a version of your component, the platform removes the associated docker build.
+*   When you remove your component entirely, the platform removes all builds of your component.
+*   We created a special tool (`remove-deleted-repos-from-docker`) to help cleanup the remnant builds. Get in touch with us to get a copy of it.
+*   As OEM customer: You must configure your own docker registry to [enable delete images option](https://docs.docker.com/registry/configuration/#delete).
+*   As OEM customer: You must know of that the deletion from docker registry does not free the memory automatically. To free the memory you would need to run the [garbage collector](https://docs.docker.com/registry/garbage-collection/) job.
+
+{% endif %}
+
+## Components
+
+### [CSV Component](/components/csv/) `3.1.3`
+
+*   **FIXED** `Emit Batch` behavior
+
+### [Woocommerce Component](/components/woocommerce/) `2.0.1`
+
+*   **ADDED** `Raw Request` action
+*   **ADDED** `Upsert Object` action
+*   **ADDED** `Webhook` trigger
+*   **REMOVED** `Create Object` and `Update Object` actions removed in favor the new `Upsert Object` action.
+*   **ADDED** metadata information to all actions.
+
+### [Paypal component](/components/paypal/) `1.3.0`
+
+*   **ADDED** `Webhook` trigger
+*   **ADDED** `Create Object` action
+*   **ADDED** `Update Object` action
+*   **IMPROVED** the input metadata in `Make a Payment` action
+
+### [Vtex component](/components/vtex/) `1.4.0`
+
+*   **ADDED** `Place Order` action
+
+### [MailChimp component](/components/mailchimp/) `1.0.3`
+
+*   **ADDED** an ability to set `Ansprache` field for `Add new Subscriber` action
+
+### Updates to multiple components
+
```

Comment:
Would be nice to ensure that these updates are linked to a benefit so the customer understands *why*

Reply:
agree, but this information is not clear to me right now. The component documentation should include the detail I hope to get this from the team by then